code/datums/stat_logger/logger.dm

```diff
@@ -0,0 +1,147 @@
+/* ._.-'~'-._.-'~'-._.-'~'-._.-'~'-._.-'~'-._. */
+/*-=-=-=-=-=-=-=STAT LOGGER BY WIRE-=-=-=-=-=-=*/
+/* '~'-._.-'~'-._.-'~'-._.-'~'-._.-'~'-._.-'~' */
+
+/// Holder datum for [/datum/stats]
+/var/global/datum/stats/stats
```

Comment:
```suggestion
var/global/datum/stats/stats
```
you also never initialize this

Reply:
One of the TODO items in the PR description is to place the initializing logic.